case-lib/lib.sh

```diff
@@ -357,6 +384,44 @@ func_mtrace_collect()
     sudo "${mtraceCmd[@]}" >& "$clogfile" &
 }
 
+func_lib_log_post_process()
+{
+    # nothing to do unless dictionary backends of Zephyr
+    # are used
+    if ! is_ipc4 && ! is_firmware_file_zephyr; then
+        return 0
+    fi
+
+    # systprint tool is available via
+    # - https://github.com/zephyrproject-rtos/mipi-sys-t
+    # - https://github.com/MIPI-Alliance/public-mipi-sys-t
+    if [ -z "$SYSTPRINT" ]; then
+        SYSTPRINT=$(command -v systprint) || {
+            dlogw 'No systprint found in PATH'
+            return 1
+        }
+    fi
+
+    local logfile="$LOG_ROOT"/mtrace.txt
+    local outfile="$LOG_ROOT"/mtrace-decoded.txt
+
+    grep -q "SYS-T RAW DATA:" $logfile || {
+        return 0
```

Comment:
Is this normal? At the very least a `dlogw` if not.

If normal then you can drop the `{}`